control-plane/csi-driver/node/src/nodeplugin_svc.rs

```diff
@@ -31,6 +32,7 @@ pub(crate) enum ServiceError {
     BlockDeviceMount { volid: String },
```

Comment:
`volid` -> `vol_id` ?

Reply:
This is existing code, but it should have an _ indeed, I'll change it.

Reply:
Thanks